live/live-root/usr/bin/live-self-update

```diff
@@ -117,17 +129,31 @@ query_reg_server() {
     info "Running query $query...."
 
     local curl_options
+    curl_options=()
```

Comment:
np: I would initialize it with the default options you are appending later

`curl options="--silent --globoff --location --retry 3 --retry-connrefused --connect-timeout 20 --max-time 10 --fail --show-error"`